config/locales/en/topical_events.yml

```
@@ -5,9 +5,9 @@ en:
     headings:
       announcements: Announcements
       consultations: Consultations
-      detailed_guides: Detailed Guides
       documents: Documents
       featured: Featured
+      guidance_and_regulation: Guidance and Regulation
```

```suggestion
      guidance_and_regulation: Guidance and regulation
```
We use sentence-case capitalisation for heading e.g. https://github.com/alphagov/collections/blob/main/config/locales/en/organisations.yml#L249

Would you mind also updating `Travel Advice` in a separate commit while you are there ;)

---

Ah, good spot. Done and added travel advice separately as a commit as you suggested 🙂